projects/distributed-tracing/src/shared/components/table/data-cell/metric/metric-table-cell-parser.ts

```diff
@@ -5,20 +5,23 @@ import { TracingTableCellType } from '../../tracing-table-cell-type';
 @TableCellParser({
   type: TracingTableCellType.Metric
 })
-export class MetricTableCellParser extends TableCellParserBase<number, number, number> {
-  public parseValue(cellData: CellData): number {
-    return Math.round(this.extractValue(cellData)!);
+export class MetricTableCellParser extends TableCellParserBase<number, number | null, number | null> {
+  public parseValue(cellData: CellData): number | null {
```

Comment:
Does it make more sense to convert any nulls to undefined? We don't generally want nulls moving through the code, so cutting them off early is good

Reply:
Absolutely. Much better.